### DISCUSSION TITLE
Semantic Tokens with components baseStyle

### DISCUSSION THREAD

**Discussion by Bowenf3** [chakra-ui/chakra-ui#5522] [Q&A]
I was hoping to use the new Semantic Tokens to replace the use of mode in the baseStyle of a components theme config.

So rather than:
`color: mode("background.100", "whiteAlpha.900")(props),`

I could define a semanticToken color:
` semanticTokens: {
    colors: {
      "background.100": {
        default: "#FDFDFF",
        _dark: "#11151c",
      }}}`

And use it something like:
`color:  "background.100",`

So that the token is referenced for default and dark mode rather than building the mode for each component?

**Comment by esitarz** [comment]
Hey @Bowenf3 not sure where you left off on this, but I figured I would just share how I solved this.

For my project, I've created a semantic token for each color in my palette. 
I prefixed my semantic tokens with `st.` so I wouldn't confuse actual theme colors with my semantic tokens:

``` 
export const semanticTokens = {
 
  colors: {
  
  "st.blackAlpha.50": {
    default: "blackAlpha.50",
    _dark: "whiteAlpha.100",
  },
  
  "st.blackAlpha.100": {
    default: "blackAlpha.100",
    _dark: "whiteAlpha.100",
  },
  
  "st.blackAlpha.200": {
    default: "blackAlpha.200",
    _dark: "whiteAlpha.200",
  }, 

  {...}
```

This helps me standardize my dark mode colors, both in the theme and as style props. 
I also like that it's based off the palette names instead of a custom name, like `background.100` because now I don't even need to remember my semantic token names-- I just drop the color I want and prefix with the `st.` to get dark mode for free:

```
const Checkbox = {
  baseStyle: {
    control: {
      borderColor: "st.blackAlpha.300",
    }
  }
}
```

or 

```
<Checkbox color="st.red.500">Checkbox</Checkbox>
